geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateDefinedIndexesCommand.java

```diff
@@ -15,13 +15,13 @@
 
 package org.apache.geode.management.internal.cli.commands;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
Please use explicit imports. Preferably have your IDE use the relevant style guide in `etc/`.

Reply:
Thanks for the tip about the style, I'll include the change in the next commit.